src/components/StudyList/__tests__/StudyTable.test.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import {MemoryRouter} from 'react-router-dom';
+import {render, cleanup} from 'react-testing-library';
+import allStudies from './allStudies.json';
+import StudyTable from '../StudyTable';
+
+afterEach(cleanup);
```

Comment:
since we call `render()` multiple times we need this to unmount those components after each test
> Unmounts React trees that were mounted with render. [Docs](https://testing-library.com/docs/react-testing-library/api#cleanup)

Reply:
Except we render it on to a new tree in every test and test against that.
Maybe we should set it up as a [global config](https://testing-library.com/docs/react-testing-library/setup#cleanup) instead?